app/src/App.vue

```diff
@@ -47,16 +49,42 @@ onBeforeMount(async () => {
         Sentry.captureException(err);
     }
 });
+
+// Wait 1 second to allow the socket connection to be established before checking the connection status
+setTimeout(() => {
+    watch(
+        isConnected,
+        () => {
+            if (!isConnected.value) {
+                useNotificationStore().addNotification({
+                    id: "offlineBanner",
```

Comment:
While this is a fine solution, another solution could be to return the `id` from `addNotification` and save that in a variable. Then it's a little easier to reference any notification instead of having to come up with unique names.